kafka-connect-s3/src/main/java/io/confluent/connect/s3/S3SinkTask.java

```diff
@@ -343,6 +348,19 @@ TopicPartitionWriter getTopicPartitionWriter(TopicPartition tp) {
     return topicPartitionWriters.get(tp);
   }
 
+  private void flushAll() {
+    log.info("Flushing local data into S3");
+
+    for (TopicPartition tp : topicPartitionWriters.keySet()) {
+      try {
+        log.info("Flushing {} local data into S3", tp);
```

Comment:
thinking about it, we don't need this msg, it would just spam, it's enough IMO just to have a single msg when starting to flush

Reply:
maybe keep it as debug level